package.json

```diff
@@ -26,7 +26,24 @@
     "vscode:prepublish": "bun run lint && bun run build",
     "verify": "sheriff verify",
     "prepublishOnly": "bun ./src/scripts/module/prepare.ts",
-    "postpublish": "git checkout package.json && git checkout README.md"
+    "postpublish": "git checkout package.json && git checkout README.md",
+    "prepare": "husky"
+  },
+  "lint-staged": {
+    "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}": [
+      "biome check"
+    ],
+    "*.{js,ts}": [
+      "bun run test"
+    ],
+    "*.svg": [
+      "bun run check",
+      "bun run check-colors",
```

Comment:
I just tried to commit an SVG icon with an invalid color and this check didn't fail when executed in this lint-staged process. Can you figure out why?

Reply:
Fixed

The problem was that the script did not return an exit code

Reply:
Fixed

The problem was that the script did not return an exit code